osmosis-1/osmosis.zone_assets.json

```diff
@@ -3502,8 +3502,10 @@
       "base_denom": "udhp",
       "twitter_URL": "https://twitter.com/dhealth_network",
       "path": "transfer/channel-75030/udhp",
-      "osmosis_verified": false,
-      "osmosis_unlisted": true,
+      "osmosis_verified": true,
+      "categories": [
```

Comment:
can you please add a listing date?

Reply:
@JeremyParish69 Does this have to be in the future ?

Reply:
It should be set to now, since it has the unlisted tag before

Reply:
`listing_date_time_utc` updated, ptal